src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -46,6 +47,7 @@ public void init() {
 
     private void registerTools() {
         new CargoConfigurator().register(LiteXpansion.getInstance());
+        new Wrench().register(LiteXpansion.getInstance());
```

Comment:
Var

Reply:
If the glass one gets merged first, what will happen to this one? Its already a variable there

Reply:
If i merge that you need to pull master and make the branches work together